state/endpoint_bindings.go

```diff
@@ -189,14 +189,15 @@ func (b *Bindings) createOp(bindings map[string]string, meta *charm.Meta) (txn.O
 // - ids of spaces assigned to endpoints.
 // - application unit count.
 // - endpoint bindings that we are currently trying to update.
-func (b *Bindings) updateOps(txnRevno int64, newMap map[string]string, newMeta *charm.Meta) ([]txn.Op, error) {
+func (b *Bindings) updateOps(txnRevno int64, newMap map[string]string, newMeta *charm.Meta, force bool) ([]txn.Op, error) {
 	var ops []txn.Op
 
 	if b.app == nil {
 		return ops, errors.Trace(errors.New("programming error: app is a nil pointer"))
 	}
 
 	useTxnRevno := len(b.bindingsMap) > 0
+	defaultSpaceID := b.bindingsMap[defaultEndpointName]
```

Comment:
Suggest new name, it is too similar to network.DefaultSpaceID.  defaultEpNameSpaceID? spaceIDOfDefaultEndpoint?